common/views/components/ModalFilters/ModalFilters.tsx

```diff
@@ -1,279 +0,0 @@
-import {
```

Comment:
I can't work out what's happening to this file. Is it being deleted and GitHub's just being a weirdo?

Reply:
We don't use it anywhere. It was part of a toggle that went stale.